MagazineLayout/Public/Types/MagazineLayoutItemSizeMode.swift

```diff
@@ -132,7 +132,7 @@ public enum MagazineLayoutItemHeightMode: Hashable {
   /// For example, if you support multiline labels or dynamic type, your height is likely not known
   /// until the Auto Layout engine resolves the layout at runtime.
   public static var dynamic: MagazineLayoutItemHeightMode {
-    .dynamic(estimatedHeight: MagazineLayout.Default.ItemHeight)
+    .dynamic(estimatedHeight: 10)
```

Comment:
I need to do some testing to make sure this doesn't break anything in the Airbnb app

Reply:
Hmm, not sure about this. I think a lower estimated height could be mean more cells are initially loaded, it's a less accurate estimation. So we could be receiving worse performance to address the animation edge case.

Does the issue only occur if the estimated height puts the original layout attributes frame intersecting the top of the collection view?

Reply:
Those are good points - I'll do some more investigation